package/python-kiwi-spec-template

```diff
@@ -391,7 +400,7 @@ done
 %fdupes %{buildroot}/srv/tftpboot
 %endif
 
-%if %{_vendor} != "debbuild"
+%if %{_vendor} != "debbuild" && %{without tftp_user_package}
```

Comment:
This actually needs to be `||`

Reply:
I have the feeling I might miss something pretty obvious, but switching the logic operator to an OR (`||`) isn't it allowing to get into the if clause any non debian build regardless of the `tftp_user_package` variable value?

Reply:
hm, you're right.

Reply:
~debian && ~tftp = debian || tftp

Is that what we want?

Reply:
I am not sure I understand your expression...

What we want in plain text is:

Execute the tftp user creation if, and only if:

1. The build is not for debian

AND

2. The build is not for Tumbleweed

The `%{without tftp_user_package}` is only active for **non** Tumbleweed (or anything higher than Leap/SLE 15.x) builds.